### ISSUE TITLE
help

### ISSUE THREAD

**Issue by nuriahmet** [ilopez/GS1Parser#5]
Hello, Friend, 010404188511271221X12Y9JNZ961721080610176600715 can you help me how to isolate this barcode ?

**Comment by davideusai** [comment]
Hello, it is a GS1 structure using following AI 01.21.17.10.
If you a put a character separator just before AI 17 you will be able to parse correctly. Without, it could be more difficult.
Have a nice day

**Comment by nuriahmet** [comment]
can you share a sample code that i can use

**Comment by davideusai** [comment]
Yes of course. Instead 010404188511271221X12Y9JNZ96@1721080610176600715

**Comment by nuriahmet** [comment]
but I want to separate it as 01,21,17,10

**Comment by davideusai** [comment]
Yes, and with a separator there you will could do that
0104041885112712 21 X12Y9JNZ96 17 210806 10 176600715
